src/test/java/org/apache/commons/crypto/benchmark/CryptoStreamBenchmark.java

```diff
@@ -0,0 +1,211 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.crypto.benchmark;
+
+import org.apache.commons.crypto.benchmark.option.BenchmarkOption;
+import org.apache.commons.crypto.benchmark.option.StreamOption;
+import org.apache.commons.crypto.conf.ConfigurationKeys;
+import org.apache.commons.crypto.stream.CryptoInputStream;
+import org.apache.commons.crypto.stream.CryptoOutputStream;
+
+import java.io.*;
+import java.util.Calendar;
+import java.util.Properties;
+import java.util.Random;
+import java.util.TimeZone;
+
+public class CryptoStreamBenchmark {
+  protected String benchMarkName;
+  protected BenchmarkOption benchmarkOption;
+  protected StreamOption streamOption;
+  byte[] inputData;
+  CryptoInputStream cryptoInputStream;
```

Comment:
I suggest not to use the member for the following variables. Localize it will help simplicity of the program.
CryptoInputStream cryptoInputStream; 
OutputStream cryptoOutputStream;  
ByteArrayInputStream inputStream;  
ByteArrayOutputStream outputStream;